pnpm-lock.yaml

```diff
@@ -1,751 +1,1003 @@
-lockfileVersion: 5.4
+lockfileVersion: '6.0'
```

Comment:
this requires at least pnpm@7.24, ideally everyone starts to use 8 at the same time though